tests/cmdline/commands/test_process.py

```diff
@@ -490,3 +395,100 @@ def test_multiple_processes(self):
         self.assertIn('No callers found', get_result_lines(result)[0])
         self.assertIn(str(self.node_root.pk), get_result_lines(result)[1])
         self.assertIn(str(self.node_root.pk), get_result_lines(result)[2])
+
+
+#@pytest.mark.skip(reason='fails to complete randomly (see issue #4731)')
+@pytest.mark.requires_rmq
+@pytest.mark.parametrize('cmd_try_all', (True, False))
+@pytest.mark.usefixtures('clear_database_before_test')
+def test_pause_play_kill(cmd_try_all):
+    """
+    Test the pause/play/kill commands
+    """
+    # pylint: disable=no-member, too-many-locals
+    import os
+    import signal
+    import subprocess
+
+    from aiida.cmdline.commands.cmd_process import process_pause, process_play, process_kill
+    from aiida.cmdline.utils.common import get_env_with_venv_bin
+    from aiida.engine.daemon.client import DaemonClient
+    from aiida.manage.configuration import get_config
+    from aiida.orm import load_node
+
+    # Add the current python path to the environment that will be used for the daemon sub process. This is necessary
+    # to guarantee the daemon can also import all the classes that are defined in this `tests` module.
+    env = get_env_with_venv_bin()
+    env['PYTHONPATH'] = ':'.join(sys.path)
+
+    profile = get_config().current_profile
+    daemon = subprocess.Popen(
+        DaemonClient(profile).cmd_string.split(),
+        stderr=sys.stderr,
+        stdout=sys.stdout,
+        env=env,
+    )
+    runner = get_manager().create_runner(rmq_submit=True)
+    cli_runner = CliRunner()
+
+    test_daemon_timeout = 5.
+    calc = runner.submit(test_processes.WaitProcess)
+    start_time = time.time()
+    while calc.process_state is not plumpy.ProcessState.WAITING:
+        if time.time() - start_time >= test_daemon_timeout:
+            raise RuntimeError('Timed out waiting for process to enter waiting state')
+
+    # Make sure that calling any command on a non-existing process id will not except but print an error
+    # To simulate a process without a corresponding task, we simply create a node and store it. This node will not
+    # have an associated task at RabbitMQ, but it will be a valid `ProcessNode` so it will pass the initial
+    # filtering of the `verdi process` commands
+    orphaned_node = WorkFunctionNode().store()
+    non_existing_process_id = str(orphaned_node.pk)
+    for command in [process_pause, process_play, process_kill]:
+        result = cli_runner.invoke(command, [non_existing_process_id])
+        import traceback
+        assert result.exception is None, ''.join(traceback.format_exception(*result.exc_info))
+        assert 'Error:' in result.output
```

Comment:
For this to still work, it does need to explicitly set the `process_state` to an active state. Now this node will never even be matched in the query in the command because you added filters on the `process_state`.